cmd/main.go

```diff
@@ -116,10 +126,12 @@ func cmd() *cobra.Command {
 		},
 	}
 
+	c.Flags().StringVar(&opts.appVersion, "app-version", "0.0.0", "AppVersion provided by the Helm Chart")
```

Comment:
I think we can remove this and just pull this from the version package https://github.com/ngrok/ngrok-operator/blob/e1fbfc88940eea5b317d7e880100bea00369198b/internal/version/version.go#L29-L31

Reply:
Does this match `.Chart.AppVersion`? I'm seeing `VERSION` in the root of the repo, but how is that updated? Is that the Release version or the App version?

Reply:
The `VERSION` file in the root of the repo is used to trigger the docker image release workflow. We also use that file as part of the ldflags to the version package. This makes sure that the binary in the docker image has the version information. While the chart's AppVersion is used as the default image tag if one is not specified, user's can overwrite this and pull an older or newer image, so just passing it through might be wrong in certain cases.